index.html

```diff
@@ -7,13 +7,22 @@
 	<title>SEO Generator</title>
 	<link rel="stylesheet" href="./assets/styles/base.css">
 </head>
+
 <body class="english">
-	<section class="form-section">
+	<main>
 		<form action="#" class="form">
```

Comment:
Hey @Jonnie-Dev, one thing we can add here is a `'onsubmit'` listener, like this:

`<form onsubmit="myFunction()">`

What that would do is run the function of our choice (the one we linked) when we click the submit button or when we press enter, without us having to set an 'onclick' to the button :) 

How do you feel about adding that in, instead of the `action="#"`? 😄

Reply:
aiit, will fix that now